src/screens/GroupInfo/index.js

```diff
@@ -182,7 +186,7 @@ const GroupInfo = () => {
         <>
           <Header isCenter onPress={() => navigation.goBack()} title={memberName()} />
           <View style={styles.lineTop} />
-          <ScrollView nestedScrollEnabled={true}>
+          <ScrollView contentContainerStyle={styles.scrollContainer} nestedScrollEnabled={true}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `contentContainerStyle` prop has been added to the `ScrollView` component. This is a good practice for controlling the layout of the child components within the ScrollView. However, ensure that the styles applied do not interfere with the expected behavior of the ScrollView.
```diff
- <ScrollView nestedScrollEnabled={true}>
+ <ScrollView contentContainerStyle={styles.scrollContainer} nestedScrollEnabled={true}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->